### PR TITLE
fix : Bottom spacing provided in feed item

### LINKED ISSUE #2477: No bottom margin in twitter feed fragment
**Actual Behaviour**

No bottom margin is present in the feed

**Expected Behaviour**

It should be present

**Screenshots of the issue**

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #2477

### CODE REVIEW

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please send screenshots whenever you make UI changes

**Comment by dreadpool2** [comment]
Well it looks like this now

File: android/app/src/main/res/layout/item_twitter_feed.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@color/feed_bg"
+    android:paddingBottom="@dimen/feed_item_profile_info_padd"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/item_twitter_feed.xml]: Use standard spacing padding

Comment (dreadpool2) [reply android/app/src/main/res/layout/item_twitter_feed.xml]: Actually this padding is equal to 10dp and the standard one is much greater than the 10dp

Comment (iamareebjamal) [reply android/app/src/main/res/layout/item_twitter_feed.xml]: No problem

**Comment by dreadpool2** [comment]
@iamareebjamal It looks like this now , should I change the padding value ?

File: android/app/src/main/res/layout/item_twitter_feed.xml [hunk 2]
```diff
@@ -3,6 +3,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@color/feed_bg"
+    android:paddingBottom="@dimen/feed_item_padding_top_bottom"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/item_twitter_feed.xml]: Don't we have common spacing values?

Comment (dreadpool2) [reply android/app/src/main/res/layout/item_twitter_feed.xml]: Do you mean the padding_medium, padding_small etc those?

Comment (iamareebjamal) [reply android/app/src/main/res/layout/item_twitter_feed.xml]: Yes

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**